platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/LibraryLoader.java

```diff
@@ -42,9 +39,9 @@ public static void setLibraryLoader(LibraryLoader libraryLoader) {
   public static void load() {
     try {
       if (!loaded) {
+        loaded = true;
```

Comment:
This will mark the library as loaded even when an exception is thrown during loading. This shouldn't have any significant impact because the map will most likely crash anyway, but it might be cleaner to set the flag afterward.

Reply:
Was doing some tests and strangely enough previous setup resulted in loading the native lib twice. This fixes it. Imo this doesn't make much sense since it's not a concurrency issue as both are called from the main thread. 

I was seeing logs as:

```
com.mapbox.mapboxsdk.testapp D/LibraryLogger: Beginning load of mapbox-gl...
com.mapbox.mapboxsdk.testapp D/LibraryLogger: Beginning load of mapbox-gl...
com.mapbox.mapboxsdk.testapp D/LibraryLogger: mapbox-gl (null) was loaded normally!
com.mapbox.mapboxsdk.testapp D/LibraryLogger: mapbox-gl (null) was loaded normally!
```

vs now:

```
com.mapbox.mapboxsdk.testapp D/LibraryLogger: Beginning load of mapbox-gl...
com.mapbox.mapboxsdk.testapp D/LibraryLogger: mapbox-gl (null) was loaded normally!
```